storage/src/lib.rs

```diff
@@ -175,6 +149,17 @@ impl Storage {
         Ok(data.to_vec())
     }
 
+    // Get the data associated with an event record.
+    pub async fn get_for_event(&self, record: &Record) -> Result<Vec<u8>, Error> {
+        // Record keys are URL encoded
+        if let Ok(decoded) = urlencoding::decode(record.key()) {
+            let data = self.bucket.get_object(decoded).await?;
```

Comment:
Is this returning an SBOM? If so, better to call `self.get` instead. Once I integrate `async-compression`, `self.get(key)` will always return uncompressed json, but `self.bucket.get_object(key)` may return compressed json.

Reply:
This is intentionally bypassing get() because it already has the 'prefix' set on the key in the event.